Documentation/install/vmware/vmware-terraform.md

```diff
@@ -166,13 +165,13 @@ $ terraform plan \
 $ terraform apply \
   ../../platforms/vmware
 ```
-Shortly after running `terraform apply` master machines will appear on Tectonic console. This change may take several minutes.  
+After running `terraform apply` master machines will appear in Tectonic Console. This change may take several minutes.  
 
-Make sure to add the new Controller nodes' IP addresses in DNS for `tectonic_vmware_controller_domain` variable or update the Load balancer to include new Controller nodes.
+Make sure to add the new Controller nodes' IP addresses in DNS for `tectonic_vmware_controller_domain` variable or update the load balancer to include new Controller nodes.
```

Comment:
Replace with: "Add the new controller nodes' IP address to the DNS record for the name set in the `tectonic_vmware_controller_domain` variable, or update the load balancer configuration with the new controller nodes."